pkg/postage/listener/listener.go

```diff
@@ -233,7 +233,7 @@ func (l *listener) Listen(ctx context.Context, from uint64, updater postage.Even
 
 	l.logger.Debug("batch factor", "value", batchFactor)
 
-	synced := make(chan error)
+	synced := make(chan error, 1)
```

Comment:
Is the buffered channel here necessary?

Reply:
No, it's not needed anymore, this was leftover from previous changes.